cypress_test/integration_tests/common/helper.js

```diff
@@ -629,7 +629,8 @@ function closeDocument(fileName, testState) {
 			return;
 		}
 
-		cy.get('#uptime')
+		// ensure the page is fully loaded & active.
+		cy.get('#uptime').should('exist')
```

Comment:
I fixed this in https://github.com/CollaboraOnline/online/pull/6209, the proper way is to use .its(text) witch waits for the text property to exist.

Reply:
I saw that - thanks ! =) what frightens me most is that I don't really see how it can not have a value - we serve the page with that - and if it's a partial page load - our cy.get() should continue dispatching until the element exists - and it should have the value '0' as it is served, and then get atomically set to something valid by the JS - without any other states - at least that's what I read; which scared me.